### ISSUE TITLE
Unable to join channels

### ISSUE THREAD

**Issue by MaxLebled** [QiCuiHub/discord-audio-pipe#12]
DAP connects successfully to the server (I see the bot being online in the members list) but is unable to join a channel.

```
Error on change_channel
Traceback (most recent call last):
  File "ui.py", line 104, in change_channel
  File "site-packages\discord\abc.py", line 1074, in connect
discord.errors.ClientException: Already connected to a voice channel.
```

**Comment by QiCuiHub** [comment]
It seems like it was caused by switching channels too quickly, and the program got into some undefined state. I've created a new release that should handle these issues.

**Comment by MaxLebled** [comment]
Getting a different error in the log now:

```
Error on change_channel
Traceback (most recent call last):
  File "ui.py", line 128, in change_channel
  File "discord\abc.py", line 1106, in connect
  File "discord\voice_client.py", line 245, in connect
  File "discord\voice_client.py", line 153, in start_handshake
  File "asyncio\tasks.py", line 449, in wait_for
concurrent.futures._base.TimeoutError

```

**Comment by MaxLebled** [comment]
Thanks very much on the very quick reply time and update! I started getting a new error after this, but it turned out to be caused by role permissions on the server... I think. Not quite sure how this happened seeing as role permissions didn't change in the past 3 weeks, but oh well :) 

For future reference to anyone googling this, this is the error I was seeing (and which I fixed by changing the bot's role permissions; not its bot permissions, those were already fine).

```
Error on change_channel
Traceback (most recent call last):
  File "ui.py", line 128, in change_channel
  File "discord\abc.py", line 1106, in connect
  File "discord\voice_client.py", line 245, in connect
  File "discord\voice_client.py", line 153, in start_handshake
  File "asyncio\tasks.py", line 449, in wait_for
concurrent.futures._base.TimeoutError
```

My best guess is that there was something being incorrectly enforced with bot role permissions and that it was invisibly fixed on the back end, but it's an extremely uninformed wild guess. But it doesn't matter now!

Thanks again for your super quick response and hanging fix 😄
